tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -197,6 +209,51 @@ export class ScalarCardDataTable {
               selectedStepData.PERCENTAGE_CHANGE =
                 (closestEndPoint.y - closestStartPoint.y) / closestStartPoint.y;
               continue;
+            case ColumnHeaderType.STEP_AT_MAX:
+              if (!closestEndPointIndex) {
+                continue;
+              }
+              selectedStepData.STEP_AT_MAX = this.getMaxValueInRange(
+                datum.points,
+                closestStartPointIndex,
+                closestEndPointIndex,
+                true
+              ).step;
+              continue;
+            case ColumnHeaderType.STEP_AT_MIN:
+              if (!closestEndPointIndex) {
+                continue;
+              }
+              selectedStepData.STEP_AT_MIN = this.getMinValueInRange(
+                datum.points,
+                closestStartPointIndex,
+                closestEndPointIndex,
+                true
+              ).step;
+              continue;
+            case ColumnHeaderType.MEAN:
+              if (!closestEndPointIndex) {
+                continue;
+              }
+              selectedStepData.MEAN = this.getMean(datum.points);
+              continue;
+            case ColumnHeaderType.REAL_CHANGE:
+              if (!closestEndPoint) {
+                continue;
+              }
+              selectedStepData.REAL_CHANGE =
+                closestEndPoint.value - closestStartPoint.value;
+              continue;
+            case ColumnHeaderType.NONSMOOTHED_START_VALUE:
```

Comment:
You don't seem to have these in the UI or in the tests. Can we remove this logic and the corresponding ColumnHeaderTypes for now?